server/server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const cors = require('cors');
+const app = express();
+app.use(cors());
+
+app.use(express.json());
+
+app.post('/', (req, res) => {
+    console.log(req.body);
+    res.send('Hello, World!');
+});
+
+app.listen(3001, () => {
+  console.log('Henna App listening on port 3001');
```

Comment:
Use a string template to print out the port, so later if you change the port you don't have to remember to change it in two places. In javascript, you can use a string template like:
```
console.log(`Henna App listening on port ${PORT}`)
```

Notice that instead of quotes, I'm using the template literal backtick `